packages/keystone/databaseAdapters/ReplicaKnexAdapter.js

```diff
@@ -54,6 +57,18 @@ class ReplicaKnexAdapter extends KnexAdapter {
 
         //override runner method
         this.knex.client.runner = (builder) => {
+            const graphqlContext = graphqlCtx.getStore()
+
+            const operationType = get(graphqlContext, ['info', 'operation', 'operation'])
```

Comment:
Is that a default path for operation definition?

Reply:
So what if user called an 'mutation' inside which there will be a query. A similar question can be asked in reverse order

Reply:
As far as I checked - yes
<img width="1115" alt="image" src="https://github.com/user-attachments/assets/fa8583d4-dfbe-4a81-b1fd-299b05a1c40e">

Reply:
Docs about resolver: https://www.apollographql.com/docs/apollo-server/data/resolvers/#resolver-arguments
Info lead to this type definitions: https://github.com/graphql/graphql-js/blob/f851eba93167b04d6be1373ff27927b16352e202/src/type/definition.ts#L891-L902
And then it leads to here: https://github.com/graphql/graphql-js/blob/16.x.x/src/language/ast.ts#L319

So it's 100% correct (default apollo implementation, not modified by KS)